portafly/public/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en" class="pf-m-redhat-font">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
```

Comment:
Do we have an icon? Otherwise remove it?

Reply:
@cntlsn do we have a fedora or something?

Reply:
Maybe this one:
https://www.redhat.com/misc/favicon.ico